cime_config/buildlib

```diff
@@ -48,7 +53,7 @@ def buildlib(bldroot, libroot, case, compname=None):
             expect(False, "ERROR in {} build {} {}".format(compname,o,e))
     else:
         logger.info("Running cmake for CDEPS")
-        srcpath = os.path.join(case.get_value("CIMEROOT"),"src","components","cdeps")
+        srcpath = os.path.join(case.get_value("SRCROOT"),"components","cdeps")
```

Comment:
This path should be set in config_files.xml, the assumption here that it will be in SRCROOT/components/cdeps
should not be made.

Reply:
As was this the case in comment for CIME - this was not done in the original CDEPS implementation - can we please discuss how this should be done and I am happy to implement it.